_pages/join/README.md

```diff
@@ -9,12 +9,22 @@ Companies and other organisations who adopt, deploy or contribute to Trusted Fir
 
 Companies joining the project at the Platinum level can vote and have a seat at Board meetings. Companies joining as General members may attend general meetings and have joint representation at Board meetings. The Community class of membership is open to universitities and other similar not-for-profit organisations.
 
-To join, the company needs to sign and return a copy of the Membership Agreement and pay an annual fee which depends on the class of membership.
+You are welcome to circulate [this slide deck](/docs/TrustedFirmwareJoinDeck_2019.pdf) within your organisation which explains more about the project.
 
 ## Next Step for Your Company to Join
 
 Request a copy of the Membership Agreement
 
+<!-- Contact Form -->
```

Comment:
Does this need to be embedded HTML?

Reply:
Sadly you can't add nice looking buttons in markdown - come to think of it - there might be a way. I'll check!